source/_integrations/fitbit.markdown

```diff
@@ -13,13 +13,43 @@ ha_integration_type: integration
 
 The Fitbit sensor allows you to expose data from [Fitbit](https://fitbit.com/) to Home Assistant.
 
+## Requirements before installation
+
 Prior to enabling the sensor, your Home Assistant instance must be enabled with SSL, using services such as [Home Assistant cloud](/cloud/) or [Duck DNS](/integrations/duckdns/) integrating Let's Encrypt. Fitbit will not allow insecure integrations.
 
+It is necessary to have configured the internal and external URLs. Settings-> System-> Network → (Home Assistant URL)
+For example something like this if you have configured Duck DNS changing to the corresponding values.
+- Internet: https://CustomName.duckdns.org:8123
+- Local Network https://192.168.x.xxx:8123
+
 <div class='note'>
 There is no need for Access from the Internet to Home Assistant for this Integration to work. Home Assistant can only be accessible from the internal Network or via VPN. But a Certificate is required in any case.
 </div>
 
-Your Fitbit account must also be registered as an Developer account at the [Fitbit Developer Portal](https://dev.fitbit.com), and have a verified email address. 
+## Create app on Fitbit
+
+Your Fitbit account must also be registered as an Developer account at the [Fitbit Developer Portal](https://dev.fitbit.com), and have a verified email address.
+
+It is the process of creating the [developer application](https://dev.fitbit.com/apps/new) in Fitbit:
+
+### Required fields
+
+**OAuth 2.0 Application Type**
+ 
+For "OAuth 2.0" it needs to be of type personal
```

Comment:
This information is also provided by the setup process.

Reply:
That might be redundant with the configuration process, but it was to avoid leaving the other callback topic alone